docker/readme.md

```diff
@@ -149,3 +154,20 @@ RAVEN_ARGS='--log-to-console'
 ##### How to set a custom config file?
 
 Mount it as a docker volume and use `--config-path PATH_TO_CONFIG` command line argument in order to use settings file from outside of server directory.
+
+##### What migration process from 5.4 image looks like?
+
+Data is stored in a different directory, and needs to be migrated or linked on the update. The container is now using the .deb package, which is well documented [here](https://github.com/ravendb/ravendb/blob/v5.4/scripts/linux/pkg/linux-packaging.md). The link describes new filesystem locations and permissions. 
```

Comment:
You might wanna use the docs link https://ravendb.net/docs/article-page/latest/csharp/start/installation/gnu-linux/deb